Lesson_4/HomeWork_4_6.py

```diff
@@ -0,0 +1,23 @@
+# Task 6. Write a program to convert currencies:
+# UAH --> USD
+# USD --> UAH
+# UAH --> euro
+# euro --> UAH
+
+USD_rate = float(38.50)
+euro_rate = float(42.00)
+UAH_rate = float(1.00)
+print('What currency do you have: USD, UAH, EURO')
+Income_currency = (input())  # getting current currency
+print('What currency do you WANT to HAVE: USD, UAH, EURO')
+Outcome_currency = (input())  # getting wanted currency
```

Comment:
outcome_currency = input("'What currency do you WANT to HAVE: USD, UAH, EURO") - this correct